### DISCUSSION TITLE
What is the abbreviation list provided by Spacy out of the box ?

### DISCUSSION THREAD

**Discussion by pulkitmehtawork** [explosion/spaCy#11196] [Help: Other Questions] [answered]
When we do sentence segmentation using spacy , we often come across abbreviations and it doesn't break them . Can you please guide how to get all abbreviation inside Spacy ?

**Comment by polm** [comment, marked as answer]
The way the tokenizer works is somewhat complicated, so there's not just a list of abbreviations anywhere, but the related data is in [`tokenizer_exceptions.py`](https://github.com/explosion/spaCy/blob/2a6ed9a40421813a97d014e9abc1ac984fb21dd5/spacy/lang/en/tokenizer_exceptions.py). That link is for English, but there are different settings for different languages.

If you want to customize the tokenizer, see the [tokenization docs](https://spacy.io/usage/linguistic-features#tokenization).
